package.json

```diff
@@ -46,7 +47,7 @@
     "less-loader": "^10.2.0",
     "mini-css-extract-plugin": "^2.6.0",
     "mock-local-storage": "^1.1.20",
-    "nightwatch": "^1.7.13",
+    "nightwatch": "^2.0.10",
```

Comment:
Can you please update nightwatch to 2.1.3 because it includes a vulnerability fix and some bug fixes.